### ISSUE TITLE
Issue with "ElementCSSInlineStyle.style": …

### ISSUE THREAD

**Issue by byterider** [mdn/content#1427]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style

#### What information was incorrect, unhelpful, or incomplete?
style property isn't readonly, according to
https://github.com/microsoft/TypeScript/issues/38838

#### Specific section or headline?
"Styles should not be set by assigning a string directly to the style property (as in elt.style = "color: blue;"), since it is considered read-only, as the style attribute returns a CSSStyleDeclaration object which is also read-only."

#### What did you expect to see?
Either an explanation _why_ it shouldn't be used, otherwise:
Per https://drafts.csswg.org/cssom/#elementcssinlinestyle, the style setter is forwarded to style.cssText, and thus accepts a string.

#### Did you test this? If so, how?
document.body.style = "background: red"; works.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/elementcssinlinestyle/style`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/api/elementcssinlinestyle/style/index.html
* Report started: 2021-01-18T11:30:13.189Z

</details>

**Comment by chrisdavidmills** [comment]
@rachelandrew , you've been doing some docs work in this area recently. Are you able to clear this one up?

**Comment by rachelandrew** [comment]
@chrisdavidmills I'll take a look

**Comment by pygy** [comment]
TL;DR: Per https://drafts.csswg.org/cssom/#elementcssinlinestyle, the style setter is forwarded to style.cssText, and thus accepts a string.

It was discussed here: https://github.com/w3c/csswg-drafts/issues/5127
